### DISCUSSION TITLE
Stack for Static Site Generator

### DISCUSSION THREAD

**Discussion by oleksis** [BlackPythonDevs/blackpythondevs#14] [Polls]
Which Stack would you select to create a [Static Site Generator](https://github.com/BlackPythonDevs/.github/discussions/1#discussioncomment-6898012)

**Comment by kjaymiller** [comment]
All the bias in the world but it would sure make me develop on it faster

**Comment by mannyanebi** [comment]
Let's see where this goes 🙌
